manifests/init.pp

```diff
@@ -110,6 +110,12 @@
   $bool_absent=any2bool($absent)
   $bool_audit_only=any2bool($audit_only)
 
+  # Hiera support
+  $hiera_sources = hiera_hash('sysctl::values', {})
+  if $hiera_sources {
```

Comment:
I expect this to return true with the default {} value, at least on Puppet 4, which is not what we want here.